docs/miscellaneous/logging.rst

```diff
@@ -0,0 +1,29 @@
+.. _logging_section:
+
+Logging
+=======
+
+In **alchemlyb**, we use :mod:`loguru` for logging. By default, the
+:mod:`loguru` will print the logging information into the `stderr`.
+
+Print to the stderr
+-------------------
+
+If you want to customise the printing to the `stderr`, you could remove the
+existing sink first ::
+
+    from loguru import logger
+    logger.remove()
+
+Then add other custom sink ::
```

Comment:
Link to a page that explains what the format does, filter etc — basically the docs for `logger.add`